packages/blade/src/components/Dropdown/docs/DropdownWithSelect.stories.tsx

```diff
@@ -488,6 +488,108 @@ export const InternalDropdownPerformance = (): React.ReactElement => {
   );
 };
 
+export const WithVirtualization = (): React.ReactElement => {
+  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
add the explicit types, looks simple enough

Reply:
😢 forgot to update these